game/player.py

```diff
@@ -5,10 +5,14 @@
 
 
 class Player:
+    """Holds current information about the player which is saved as a global variable"""
+
     team: List[Pokemon] = []
 
     def __init__(self):
-        if (len(self.team) < 1):
+        """Fills "Team" array with a random Pokemon from PokeApi and saved as a global variable"""
```

Comment:
Minor: don't capitalize references to variables.
`Fills the player's team with a random Pokemon from PokeApi`
The `Player` class doesn't actually do any of the saving to the global `PLAYER` variable, so I wouldn't include that.